core/src/avm2/object.rs

```diff
@@ -229,6 +234,16 @@ pub trait TObject<'gc>: 'gc + Collect + Debug + Into<Object<'gc>> + Clone + Copy
         self.base().get_property_local(name, activation)
     }
 
+    /// Same as get_property_local, but constructs a public Multiname for you.
+    fn get_string_property_local(
```

Comment:
BTW, I think all three of these `*_string_property_local` could be `#[no_dynamic]`?